websites/N/Nexon Logistics/presence.ts

```diff
@@ -0,0 +1,174 @@
+				case document.location.pathname.includes("/events/"):
+					presenceData.details = "Viewing an Event";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h4")[0].textContent;
+					presenceData.buttons = [{ label: "View Event", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/members"):
+					presenceData.details = "Viewing Members Page";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h4")[0].textContent;
+					presenceData.buttons = [{ label: "View Members", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/jobs"):
+					presenceData.details = "Browsing Jobs";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h4")[0].textContent;
+					presenceData.buttons = [{ label: "View Jobs", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/downloads"):
+					presenceData.details = "Browsing Download Section";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h2")[0].textContent;
+					presenceData.buttons = [
+						{ label: "View Downloads", url: document.URL },
+					];
+					break;
+				case document.location.pathname.includes("/vtcrules"):
+					presenceData.details = "Browsing Knowlege Base";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h2")[0].textContent;
+					presenceData.buttons = [
+						{ label: "View VTC Rules", url: document.URL },
+					];
+					break;
+				case document.location.pathname.includes("/patreons"):
+					presenceData.details = "Viewing A Page";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h1")[0].textContent;
+					presenceData.buttons = [
+						{ label: "View Patreons", url: document.URL },
+					];
+					break;
+				case document.location.pathname.includes("/faqs"):
+					presenceData.details = "Viewing A Page";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h2")[0].textContent;
+					presenceData.buttons = [{ label: "View FAQs", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/news"):
+					presenceData.details = "Viewing A Page";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h2")[0].textContent;
+					presenceData.buttons = [{ label: "View News", url: document.URL }];
+					break;
+				case document.location.pathname.includes("/loa"):
+					presenceData.details = "Applying For";
+					presenceData.state =
+						document.querySelectorAll<HTMLHeadingElement>("h2")[0].textContent;
+					presenceData.buttons = [{ label: "View LOA", url: document.URL }];
+					break;
+				default:
+					presenceData.details = "Browsing Nexon Drivers Hub";
+			}
+			break;
+		case "status.nexonlogistics.com":
+			switch (true) {
+				case document.location.pathname === "/":
+					presenceData.details = "Browsing Nexon Server Status";
+			}
```

Comment:
There is no need to use a switch statement for this, there is only 1 case. Use an if statement instead.

Reply:
I have made changed but please check whether its good

Reply:
Since it is still just a single case and a default, it may be better to use an if/else statement instead.